readme.md

```diff
@@ -83,29 +74,28 @@ pify(request, {multiArgs: true})('http://sindresorhus.com').then(result => {
 
 Type: `array`
 
-*Works for `pify.all()` only.*
-
 Pick which methods in a module to promisify. Remaining methods will be left untouched.
 
+You can specify either `strings` or `regular expressions` as method names.
+
 ##### exclude
 
-Type: `array`
+Type: `array`  
+Default: `[/^.+Sync$/]`
 
-*Works for `pify.all()` only.*
+Pick which methods in a module **not** to promisify. Methods with names ending with `'Sync'` are excluded by default.
 
-Pick which methods in a module **not** to promisify.
+You can specify either `strings` or `regular expressions` as method names.
 
 ##### excludeMain
 
 Type: `boolean`  
 Default: `false`
 
-*Works for `pify.all()` only.*
-
 By default, if given `module` is a function itself, this function will be promisified. Turn this option on if you want to promisify only methods of the module.
 
 ```js
-const pify = require('pify');
+import pify from 'pify';
```

Comment:
I've never used `import` in my readmes. What you've looked at was probably my test files, which does use `import`, since I use AVA for tests, which comes with builtin full support for ES2015 (and `import`).